src/python/pants/backend/python/dependency_inference/scripts/BUILD

```diff
@@ -0,0 +1,4 @@
+resources(name="resources", sources=["*.py"])
```

Comment:
This is totally fine, but things may be clearer if you use a resource and python_source target instead because there's only one file. For example, you won't need to use a file address.